deep_quoridor/src/agents/alphazero/nn_evaluator.py

```diff
@@ -63,18 +63,27 @@ def evaluate(self, game: Quoridor):
         assert np.any(policy_masked > 0), "Policy is all zeros"
         assert np.isfinite(policy_masked).all() and np.isfinite(value), "Policy or value is non-finite"
 
+        # If the game was originally rotated, rotate the resulting back to player 2's perspective
+        if is_board_rotated:
+            policy_masked = policy_masked[self.rotated_action_mapping]
```

Comment:
Hmmm.... is the rotated_action_mapping that maps from an action on a regular board to an action on a rotated board the same as the that inverts that process? Rotating a board twice does put the board back in its originally orientation, but is the same true for rotating actions in this way...... hmmm.....

Reply:
Hmm ... I tried it with B3W0 and it seemed to work, but that only tested the moves part, not wall placement ...